src/mongo/delegates/Updates.class.php

```diff
@@ -467,10 +468,10 @@ protected function applyChangeSet(\Tripod\ChangeSet $cs, $originalCBDs, $context
 
                 $doc = $this->getDocumentForUpdate($subjectOfChange, $contextAlias, $originalCBDs);
 
-                $targetGraph = new MongoGraph();
-                $targetGraph->add_tripod_array($doc);
+//                $targetGraph = new MongoGraph();
+//                $targetGraph->add_tripod_array($doc);
 
-                if(!$targetGraph->has_triples_about($subjectOfChange))
+                if(empty($doc))
                 {
                     $upsert = true;
```

Comment:
I realize this isn't new here, but I think this is a bug: this could potentially set ` upsert=true ` for every change, couldn't it?